### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by eggaskin**
code is to be tested, then squashed and merged. then we can use master and develop as intended

### CODE REVIEW

**Review by Ippyb — APPROVED** [review]

**Review by seasew — APPROVED** [review]
Looks good 👍

File: src/main/deploy/config.txt [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# drive motors CAN IDs
```
Comment (e3l) [inline src/main/deploy/config.txt:1]: can be cleared

File: src/main/java/frc/robot/Constants.java [hunk 2]
```diff
@@ -5,11 +5,28 @@
...
- * <p>It is advised to statically import this class (or one of its inner classes) wherever the
- * constants are needed, to reduce verbosity.
+ * <p>
+ * It is advised to statically import this class (or one of its inner classes)
+ * wherever the constants are needed, to reduce verbosity.
  */
-public final class Constants {}
+public final class Constants {
+    public static final class TankConstants {
+        public static final int fLeftMotorPort = 15;
```
Comment (e3l) [inline src/main/java/frc/robot/Constants.java:19]: can be ALL_CAPS

**Review by e3l — APPROVED** [review]
lgtm

**Pull request merged**